src/test/java/de/tum/in/www1/artemis/FileIntegrationTest.java

```diff
@@ -412,7 +392,7 @@ private void adjustReleaseDateToFuture(Lecture lecture) {
     }
 
     @Test
-    @WithMockUser(username = TEST_PREFIX + "tutor1", roles = "TA")
+    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     void testGetLecturePdfAttachmentsMerged_correctOrder() throws Exception {
         Lecture lecture = createLectureWithLectureUnits();
 
```

Comment:
line 405: remove the comment `// The test setup needs at least TA right for creating a lecture with files.`

Reply:
Wait, I'm a bit confused now. We still execute the `createLectureWithLectureUnits` as an instructor (instead of a tutor) and then later switch to a student for the testing. So this comment is still valid and stay there imo, just replace tutor with instructor.

Same above since there is also a user switch mid test.

Reply:
Do you think, though, the comment has any value?

Reply:
I don't think the comment is really needed, I recommended to remove it since you already removed a similar one in a different test. If you decide to re-add it, that's also ok

Reply:
I remember adding a test here and I was confused why testing with a student account did not work. When setting up data we normally don't use the endpoints, so having to use an instructor for the setup here is not clear and imo is worth a comment.